sdk/communication/azure-communication-chat/pom.xml

```diff
@@ -38,6 +38,7 @@
   <properties>
     <jacoco.min.linecoverage>0.70</jacoco.min.linecoverage>
     <jacoco.min.branchcoverage>0.45</jacoco.min.branchcoverage>
+    <jacoco.skip.coverage.check>true</jacoco.skip.coverage.check>
```

Comment:
I saw you change the default value for other packages, do you need to change it here as well?